src/Storage/Storage.Management/help/New-AzStorageAccountSASToken.md

```diff
@@ -75,6 +75,7 @@ Accept wildcard characters: False
 
 ### -ExpiryTime
 Specifies the time at which the shared access signature becomes invalid.
+To get a **DateTime** object, use the Get-Date cmdlet.
```

Comment:
User can just give the expire time in string like `-ExpiryTime "2021-12-12 20:40"`, so user don't need always use `Get-Data`

Reply:
Updated.